playbooks/centurion_teams.yaml

```diff
@@ -0,0 +1,244 @@
+- name: Centurion ERP Teams Setup
```

Comment:
use two commits for these playbooks as there are two features added.

Reply:
Ack